kombu/asynchronous/http/base.py

```diff
@@ -253,5 +257,10 @@ def on_header(self, headers, line):
     def __enter__(self):
         return self
 
-    def __exit__(self, *exc_info):
+    def __exit__(
+        self,
+        exc_type: type,
+        exc_val: Exception,
+        exc_tb: 'TracebackType'
```

Comment:
Ref.: https://github.com/python/typeshed/blob/354787f6e0037f7b8d37d11eb851b8f4e0931a42/stdlib/contextlib.pyi#L187-L189
```suggestion
        exc_type: Optional[Type[BaseException]],
        exc_val: Optional[BaseException],
        exc_tb: Optional['TracebackType']
```